data/json/recipes/recipe_others.json

```diff
@@ -5330,6 +5330,41 @@
     ]
 },{
   "type" : "recipe",
+  "result": "keg_steel",
+  "category": "CC_OTHER",
+  "subcategory": "CSC_OTHER_CONTAINERS",
+  "skill_used": "fabrication",
+  "difficulty": 8,
+  "time": 164062,
+  "reversible": false,
+  "autolearn": true,
+  "qualities":[
+    {"id":"HAMMER","level":3,"amount":1},
+    { "id": "CHISEL", "level": 3 }
+  ], "tools": [
+     [
+      ["tongs", -1]
+     ],
+     [
+      ["anvil", -1]
+     ],
+     [
+      ["crucible", -1],
+      ["crucible_clay", -1]
+     ],
+     [
+      ["forge", 160], [ "oxy_torch", 25]
+     ]
+    ],
+  "components":
+    [
+     [
+     [ "steel_lump", 13 ],
+     [ "steel_chunk", 52 ]
+     ]
+    ]
```

Comment:
Having read through #18347, I'd like to bring up that I'm actually in a similar boat to @kilozombie, in that I'm editing directly to the remote, rather than using GitHub locally on my machine. In my case, I can't actually operate GitHub locally (56k dialup does that to you), so I have to make the changes in a textpad, then copy-paste them into my fork online (after testing and making sure I've fixed anything I broke).

As a complete novice in anything higher level than JSON, which files would I need to download to make the `format.pl` script function offline on Windows 7, and would it be breaking anything to manually update the whitelist?